api/Blob.json

```diff
@@ -310,10 +310,10 @@
               "version_added": "54"
             },
             "safari": {
-              "version_added": false
+              "version_added": "14.1"
```

Comment:
Is it worth adding a note that while it did ship in 14.1, it shipped with a bug that causes it to crash if the Blob is too big? https://bugs.webkit.org/show_bug.cgi?id=220776#c8

Reply:
@lukewarlow this PR is getting really big, so I'd suggest sending a separate PR with your suggested note text, so that can be reviewed in isolation. (Creating that PR now would result in conflicts if this is merged, but they'll be easy to resolve.)